app/holds.js

```diff
@@ -5,6 +5,7 @@ const getHolds = async () => {
   const { payload } = await get('/payment-holds')
   return payload.paymentHolds?.filter(x => x.dateTimeClosed == null).map(x => {
     x.dateTimeAdded = moment(x.dateTimeAdded).format('DD/MM/YYYY HH:mm')
+    if (x.holdCategorySchemeName === 'SFI') x.holdCategorySchemeName = 'SFI22'
```

Comment:
I'd suggest using the `{}` syntax just to be consistent with the rest of the code, plus the addition you've made below.